### PR TITLE
A few small curator adjustements

### PR DESCRIPTION
**Pull request by stkem**
see individual commits for specific changes
@yingjieMiao @lawzlo

### CODE REVIEW

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/RecommendationRetrievalCommander.scala [hunk 1]
```diff
@@ -53,7 +55,11 @@ class RecommendationRetrievalCommander @Inject() (db: Database, uriRecoRepo: Uri
   }
 
   def topPublicRecos(): Seq[RecoInfo] = {
-    db.readOnlyReplica { implicit session => publicFeedRepo.getByTopMasterScore(1000) }.sortBy(-1 * _.updatedAt.getMillis).take(10).map { reco =>
+    val candidates = db.readOnlyReplica { implicit session =>
+      publicFeedRepo.getByTopMasterScore(1000)
+    }.sortBy(-1 * _.updatedAt.getMillis).take(100)
+
+    Random.shuffle(candidates).take(10).map { reco =>
```
Comment (lawzlo) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/RecommendationRetrievalCommander.scala:62]: Why we shuffle here? Do we want to show recos to user from highest to lowest interests so they will potentially read more?

Comment (stkem) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/RecommendationRetrievalCommander.scala:62]: This is just to make the public feel a bit more dynamic, making Kifi look more alive to new users.

**Comment by stkem** [comment]
I just wanted to get some dynamism into the public feed, which can feel very static (there is no UI for the user to get more items). While sampling based on the score would indeed be better, I don't think it's worth the effort in this case.

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/RecommendationRetrievalCommander.scala (hunk 1 above)
Comment (lawzlo) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/RecommendationRetrievalCommander.scala:62]: I see. :+1:

**Comment by stkem** [comment]
I've added some recommendation score cutoff adjustments to this PR

**Comment by yingjieMiao** [comment]
cool. Maybe we can consolidate all these cutoffs in one object, like SearchConfig does. So we don't have to jump around the code to modify those numbers.

**Comment by stkem** [comment]
Good idea. 
Need to get this out to get some recos to the people who complained, but we should definitely do this. Will also allow us to have different configs for different people.

**Pull request merged**